ideascube/models.py

```diff
@@ -55,6 +55,19 @@ def create_superuser(self, serial, password, **extra):
         user.save(using=self._db)
         return user
 
+    def get_queryset(self):
+        return super().get_queryset().exclude(serial='__system__')
+
+    def get_queryset_unfiltered(self):
+        return super().get_queryset()
+
+    def all(self, really=False):
```

Comment:
Not a big fan of the kwarg pattern that changes a default Django method.
What about a separate method (`unfiltered`?) instead?